electron/src/main.ts

```diff
@@ -207,6 +207,9 @@ const showMainWindow = (mainWindowState: WindowStateKeeper.State) => {
     setTimeout(() => main.show(), 800);
   }
 
+  main.on('focus', () => systemMenu.createShortcuts());
```

Comment:
Let's call it `registerShortcuts` and `unregisterShortcuts`.

Reply:
I believe we also don't need to create new functions here and can just use
```main.on('focus', systemMenu.registerShortcuts);```